app/src/nest/communities/websocket/communities.handler.ts

```diff
@@ -47,6 +48,18 @@ export function registerCommunitiesHandlers(
   ): Promise<void> {
     _logger.debug(`Handling community create event`)
     try {
+      if (config.socket.data.verifiedCaptcha !== true) {
```

Comment:
Since the captcha is only used for creating the community we might want to make verification a part of the creation process (ie adding the captcha value to the create community request and verifying here).  This relies on the honor system so if someone wanted to get around the captcha they could just pass true for this field.

Another option would be to cache the verification status for a given socket connection.